build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+app_id="moe._999eagle.OkunaDesktop"
+
+mkdir -p "/app/opt"
+mkdir -p "/app/bin"
+mkdir -p "/app/share/applications"
+mkdir -p "/app/share/metainfo"
+
+echo "copying files"
+
+# copy main okuna files
+mv okuna-desktop /app/opt/
+
+# set up entry point
+mv okuna-desktop.sh /app/bin/
+rm /app/opt/okuna-desktop/okuna-desktop.sh
+
+# set up desktop entry
```

Comment:
As you are the upstream developer, is there any reason why these changes can't happen in code repository instead? Especially the icon part. `Exec=` should be fixed automatically by flatpak-builder.

Reply:
I thought it'd be more useful to do it like this, but I can also rely on `rename-icon=` in the manifest. Will change that.